config-windows.md

```diff
@@ -0,0 +1,80 @@
+# Windows-specific Container Configuration
+
+This document describes the schema for the [Windows-specific section](config.md#platform-specific-configuration) of the [container configuration](config.md).
+The Windows container specification uses APIs provided by the Windows Host Compute Service (HCS) to fulfill the spec.
+
+
+#### Memory
+
+`memory` is used to set limits on the container's memory usage.
```

Comment:
The existing spec is not very consistent about this, but I'd like to see an OPTIONAL dropped into this sentence.  Maybe:

> `memory` is an OPTIONAL configuration for the container's memory usage.